testutil/integration/example_test.go

```diff
@@ -63,7 +64,10 @@ func Example() {
 		logger,
 		keys,
 		encodingCfg.Codec,
-		authModule, mintModule,
+		map[string]appmodule.AppModule{
+			"auth": authModule,
```

Comment:
are we hardcoding this intentionally ("auth") ?

Reply:
This is because `appmodule.AppModule` do not have names. Or do you mean why are we not using the auth module types for getting the name?